Example-iOS/Podfile

```diff
@@ -0,0 +1,9 @@
+source 'https://github.com/CocoaPods/Specs.git'
+
+target 'VOKBenkode', :exclusive => true do
```

Comment:
Because that's the way the podfile is set up when you `pod lib create`?  I don't know that it makes a difference...  Actually, I'm not sure that the pod would be added to the test target if it were just added in general, but I'm not sure that would matter.